### PR TITLE
Update Debian Installation in Makefile

### PR DESCRIPTION
**Pull request by nuehm-arno**
install-deb-arm64 depended on prebuild-package-deb-arm64 - installation for testing purpose is now possible directly after building

### CODE REVIEW

**Comment by nuehm-arno** [comment]
And changes were made regarding `go get` in golang v1.16 - `env GO111MODULE=off` was introduced, so that the `go` command ignores `go.mod` files and runs in `GOPATH` mode.

File: Makefile [hunk 1]
```diff
@@ -234,21 +234,23 @@ build-package-deb-arm64:
...
+	@sudo cp $(CURRENT_DIR)/deb/axolotl.desktop $(DESTDIR)$(SHARE_PREFIX)/applications/
+	@sudo cp $(CURRENT_DIR)/deb/axolotl.png $(DESTDIR)$(SHARE_PREFIX)/icons/hicolor/128x128/apps/
 	@sudo cp $(CURRENT_DIR)/deb/axolotl.sh /etc/profile.d
 	@bash -c "source /etc/profile.d/axolotl.sh"
 	@echo "Installation complete"
 
 uninstall-deb-arm64:
-	@sudo rm -rf $(DESTDIR)$(SHARE_PREFIX)/axolotl
+	@pkill axolotl
+  @sudo rm -rf $(DESTDIR)$(SHARE_PREFIX)/axolotl/
```
Comment (nanu-c) [inline Makefile:253]: the whitespace is wrong here

Comment (nuehm-arno) [reply Makefile:253]: True! I wonder, how that happenend... must be magic.

Comment (nuehm-arno) [reply Makefile]: Should be good now.

**Comment by nanu-c** [comment]
Can you rebase this mr?

**Comment by nuehm-arno** [comment]
Of course, later in the evening.

**Comment by nuehm-arno** [comment]
Should be good now, thanks for your patience.

**Review by nanu-c — APPROVED** [review]

**Pull request merged**